Automattic/wp-calypso
# Fix the alignment of domain suffix for Professional Emal

#### Changes proposed in this Pull Request

When purchasing a new Professional Email subscriptions, the @ sign in @your-domain.tld should always appear to the left of the domain, but shows to the right in RTL languages. This fix wraps it in LRM/PDF characters to force the text to appear as LTR and makes sure that the input field suffix is always to the right of the email address.

Before:
<img width="887" alt="Image 2021-10-01 at 5 19 02 PM" src="https://user-images.githubusercontent.com/7847633/135645414-90667e0c-d3c6-48f8-8f62-3af95063c28e.png">

After:
<img width="887" alt="Image 2021-10-01 at 5 26 58 PM" src="https://user-images.githubusercontent.com/7847633/135646584-f59938d9-8584-47e2-8e3f-c7deed0175a0.png">

#### Testing instructions

Testing instructions

1. Change your language in Calypso to Arabic or Hebrew.
2. Checkout the branch and navigate to `http://calypso.localhost:3000/email/your-domain-name` for a site with a domain or `http://calypso.localhost:3000/domains/add/some-very-random-domain-name.uk/email/your-site-name.wordpress.com` if you don't have a domain.
3. Confirm that all the @your-domain-name strings appear correctly on the page for Titan/GSuite forms.
4. Change the language to an LTR locale and test for regression issues.

Fixes 312-gh-Automattic/i18n-issues
